imageroot/actions/create-module/02create-first-configuration

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+# first build of the configuration
+
+expand-configuration
```

Comment:
because I want that crowdsec know that it exists first a configuration in /etc/crowdsec, else at the first start, everything is removed by the docker initscript

see

https://github.com/crowdsecurity/crowdsec/blob/master/docker/docker_start.sh#L25

We erase all previous configuration done at the first start